plugins/uefi-recovery/README.md

```diff
@@ -0,0 +1,16 @@
+UEFI Support
+============
+
+Introduction
+------------
+
+Some devices do not include the ESRT table in old firmware versions.
```

Comment:
I think you should make it clearer that this is a firmware bug.  I really don't want to see vendors coming here and thinking they could just make a quirk for their broken firmware because the infrastructure to support the broken fimware already exists.

Reply:
Something like:

    Some devices have firmware bugs which mean they do not include a valid ESRT
    table in old firmware versions.